ubuntu/Dockerfile

```diff
@@ -6,14 +6,10 @@ LABEL maintainer="Hive Solutions <development@hive.pt>"
 ARG DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update &&\
-    apt-get install -y -q --no-install-recommends sudo bash wget git openssl unzip ca-certificates dirmngr apt-transport-https python2 python3 python3-distutils &&\

```

Comment:
@joamag python2 was removed due to its incompatibility with libssl3 on Ubuntu Noble